<ui version="4.0" >
 <class>ProjectConverterDialog</class>
 <widget class="QDialog" name="ProjectConverterDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Project Converter</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="projectNameLable" >
     <property name="text" >
      <string>Project XXX (no translation needed)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="conversionInfoText" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>5</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="taskLable" >
     <property name="text" >
      <string>Conversion information</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="taskTextBrowswer" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="progressLable" >
     <property name="text" >
      <string>Conversion progress</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startButton" >
       <property name="text" >
        <string>Start conversion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopConversionButton" >
       <property name="text" >
        <string>Stop conversion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadProjectButton" >
       <property name="text" >
        <string>Load Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ProjectConverterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>388</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel" >
     <x>449</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>ProjectConverterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>103</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel" >
     <x>10</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
